themes/bootstrap3/js/common.js

```diff
@@ -679,13 +679,10 @@ function bulkFormHandler(event, data) {
     VuFind.lightbox.alert(VuFind.translate('bulk_noitems_advice'), 'danger');
     return false;
   }
-  if (event.originalEvent !== undefined) {
-    let limit = event.originalEvent.submitter.dataset.itemLimit;
+  if (event.submitter !== undefined) {
+    let limit = event.submitter.dataset.itemLimit;
     if (numberOfSelected > limit) {
-      VuFind.lightbox.alert(
-        VuFind.translate('bulk_limit_exceeded', {'%%count%%': numberOfSelected, '%%limit%%': limit}),
-        'danger'
-      );
+      alert('Bulk limit exceeded: ' + numberOfSelected + '. Limit is: ' + limit);
```

Comment:
This change will cause some feature regressions: it breaks VuFind's built-in internationalization functionality and it replaces a message displayed as part of the page's HTML with a generic browser pop-up message, which is a less pleasant user experience. If this change was necessary to get tests to pass, I think we'll need to figure out why so we can restore the functionality that is otherwise lost. @crhallberg and I will both be in the office tomorrow if it would be helpful to take a look at this together.

Reply:
It worked without changing the previous code. Reverted to lightbox alert.